interface-spec.md

```diff
@@ -66,15 +68,15 @@
 
 **Properties:**
 
-- `Node .graph` the named graph, which is an IRI, a BlankNode or NodeVariable.
+- `Node .why` the provenence, which may be a named node represenitng a document on the web which has this graph, or a BlankNode or NodeVariable. Also extednable for the results of infreence.
 
 ### DataFactory
 
 **Methods:**
 
-- `.iri(String iri)` returns a new instance of IRI.
-- `.blankNode()` returns a new instance of BlankNode.
+- `.sym(String iri)` returns a new instance of IRI.
+- `.bNode()` returns a new instance of BlankNode.
```

Comment:
The problem with "IRI" is that the node is NOT an IRI.  It has one.   A node which is labelled by an IRI may be referred to in the RDF spec loosely as an IRI but it is not an IRI, an IRI is a string.   I am much less concerned about methods than properties, as is easy to add both methods.